deepchem/models/torch_models/tests/test_grover_layers.py

```diff
@@ -6,22 +6,6 @@
     pass
 
 
-@pytest.fixture
-def grover_graph_attributes():
```

Comment:
Do we no longer need this fixture?

Reply:
The fixture was moved to `conftest.py` in #3334 as it is being used be more than one test file.